include/bluetooth/hci_raw.h

```diff
@@ -21,23 +21,23 @@
 extern "C" {
 #endif
 
-#if defined(CONFIG_BT_CTLR_TX_BUFFER_SIZE)
-#define BT_L2CAP_MTU (CONFIG_BT_CTLR_TX_BUFFER_SIZE - BT_L2CAP_HDR_SIZE)
-#else
-#define BT_L2CAP_MTU 65 /* 64-byte public key + opcode */
-#endif /* CONFIG_BT_CTLR */
+#define _BT_ACL_BUF_SIZE(len) (BT_BUF_RESERVE + \
+				BT_HCI_ACL_HDR_SIZE + \
+				(len))
 
 /** Data size needed for ACL buffers */
-#define BT_BUF_ACL_SIZE BT_L2CAP_BUF_SIZE(BT_L2CAP_MTU)
+#ifdef CONFIG_BT_HCI_ACL_DATA_SIZE
+#define BT_BUF_ACL_SIZE _BT_ACL_BUF_SIZE(CONFIG_BT_HCI_ACL_DATA_SIZE)
+#else
+#define BT_BUF_ACL_SIZE _BT_ACL_BUF_SIZE(27)
```

Comment:
CONFIG_BT_HCI_ACL_DATA_SIZE should always be defined, since it only depends on CONFIG_BT_HCI_RAW, and it has the same default value of 27.
Removing this else would mean that the value always comes from Kconfig, which would be cleaner.